pyproject.toml

```diff
@@ -30,6 +30,7 @@ graphql-core = {version = "^3.0.0"}
 asgiref = {version = "^3.2",optional = true}
 flask = {version = "^1.1",optional = true}
 typing_extensions = "^3.7.4"
+opentracing = "^2.3.0"
```

Comment:
looks like opentracing will become opentelemetry, would be ok in trying to use that instead? no worries if not, this is great starting point anyway!

I didn't realise opentelemetry was a thing :)

Reply:
I could try OpenTelemetry, but would like to go with opentracing for now. May be later this month!?